components/BattleScreen/BattleScreen.js

```diff
@@ -1,11 +1,66 @@
-import React, { useState } from "react";
 import styled from "styled-components";
+
+import React, { useState } from "react";
 import PlayerPokemon from "./Pokemon/PlayerPokemon";
 import PlayerState from "./PlayerState/PlayerState";
 import EnemyPokemon from "./Pokemon/EnemyPokemon";
 import EnemyState from "./EnemyState/EnemyState";
 import Menu from "./Menu/Menu";
-import VictoryMessage from "./VictoryMessage/VictoryMessage";
+import VictoryMessage from "./Message/VictoryMessage";
+import LoserMessage from "./Message/LoserMessage";
+
+export default function BattleScreen() {
+	const [enemyHealth, setEnemyHealth] = useState(120);
+	const [playerHealth, setPlayerHealth] = useState(120);
+	const [victory, setVictory] = useState(false);
+	const [isDisabled, setIsDisabled] = useState(false);
+	const [isEnemyDefeated, setIsEnemyDefeated] = useState(false);
+
```

Comment:
In a game like this, you'll have to handle alot of states. You might consider using a store of for example the zustand library or yotai.